capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -181,37 +181,37 @@ class ClubPage extends Component {
           {this.state.meetings.map(m => <MeetingCard meeting={m} deleteMeeting={this.deleteMeeting} />)}
         </CardDeck>
       </>;
-      
-    const bookTile = this.state.book.authors 
-      ? <BookSearchTile 
-        book={this.state.book} 
-        bookLists={this.props.bookLists} 
-        updateBookLists={this.props.updateBookLists} 
+
+    const bookTile = this.state.book.authors
+      ? <BookSearchTile
+        book={this.state.book}
+        bookLists={this.props.bookLists}
+        updateBookLists={this.props.updateBookLists}
         location={'search'}
       />
       : <span className='block'> No book yet! </span>
-    const owner = this.state.owner 
-      && <UserCard 
-            removeMember={this.removeMember} 
-            club={this.state.club} 
-            user={this.state.owner}
-            updateFriendsList={this.props.updateFriendsList}
-          />;
-    const members = this.state.members.length > 0 
-      ? this.state.members.map(m => 
-        <UserCard 
-          key={m.id} 
-          user={m} 
-          club={this.state.club} 
-          removeMember={this.removeMember} 
```

Comment:
If you're going to keep working on this project (or for future projects), investing in setting up an auto-formatter or at least a good linter should prevent diffs like these. Always nice to not have to think about code formatting.

Reply:
Good idea, the built in Gitpod formatter can be finicky.